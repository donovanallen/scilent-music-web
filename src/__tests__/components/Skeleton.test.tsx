import { render } from '@testing-library/react';
import React from 'react';

import Skeleton from '@/components/Skeleton';

describe('Skeleton', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Skeleton />);
    expect(asFragment()).toMatchSnapshot();
  });
});
