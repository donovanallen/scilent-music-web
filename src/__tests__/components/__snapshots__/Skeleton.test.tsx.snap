// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton renders correctly 1`] = `
<DocumentFragment>
  <div
    class="animate-shimmer bg-[#f6f7f8]"
    style="background-size: 700px 100%; background-repeat: no-repeat;"
  />
</DocumentFragment>
`;
